metrics_sdk/lib/opentelemetry/sdk/metrics/configuration_patch.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      # The ConfiguratorPatch implements a hook to configure the metrics
+      # portion of the SDK.
+      module ConfiguratorPatch
+        private
+
+        # The metrics_configuration_hook method is where we define the setup process
+        # for metrics SDK.
+        def metrics_configuration_hook
+          OpenTelemetry.meter_provider = Metrics::MeterProvider.new(resource: @resource)
```

Comment:
The @resource instance var will be available in the configurator, it's a little awkward as were developing this as a separate gem, but once we have stabilized the metrics api/sdk it will get merged into the SDK/API and this patch stuff will go away.